packages/cloud-toolkit/src/build-command.ts

```diff
@@ -27,12 +28,15 @@ const logger: Logger = {
 };
 
 export default function buildCommands() {
+  const { version } = JSON.parse(fsSync.readFileSync('package.json', 'utf-8'));
```

Comment:
this doesn't work
You need something like that:
```suggestion
  const { version } = JSON.parse(
    fsSync.readFileSync(path.join(__dirname, 'package.json'), 'utf-8'),
  );
```